recipe/meta.yaml

```diff
@@ -44,11 +44,12 @@ requirements:
     - libuv 1.44    # [not win]
     - openssl 1.1.1 # [not win]
     - zlib 1.2      # [not win]
+    - icu {{ icu }}
   run:
     # icu min version is 69. It won't compile without this minumum version.
     # until the distribution gets updated to a new icu, internationalization is disabled. 
-    - icu >=58,<59.0a0        # [not (win or s390x or aarch64 or (osx and arm64))]
-    - icu >=68,<69.0a0      # [s390x or aarch64 or (osx and arm64)]
+    - icu >=58,<=73.1      # [not (win or s390x or aarch64 or (osx and arm64))]
```

Comment:
you need to update the lower bounds here as well, and the upper bound isn't correct. Have a look at what it was before, and how the host version related to the runtime version range. Also, try and figure what the explanation might be for that. (there's a comment in the icu recipe that relates to this. Also, it would be useful to read up on `run_exports` and `pin_subpackage` in the [conda-build docs](https://docs.conda.io/projects/conda-build/en/latest/index.html))